Cognite.Simulator.Utils/ConnectorBase.cs

```diff
@@ -51,6 +55,7 @@ public abstract class ConnectorBase
             _simulatorSequenceIds = new Dictionary<string, string>();
             _logger = logger;
             _config = config;
+            LastLicenseCheckResult = ShouldLicenseCheck() ? "Not checked yet" : "License check disabled";
```

Comment:
ShouldLicenseCheck is virtual and prob won't be called from here, maybe make it just "protected"?